src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.tsx

```diff
@@ -203,15 +203,7 @@ export class DataSourceAggregatedView extends React.Component<
           anchorPosition="downLeft"
         >
           <EuiContextMenuPanel>
-            <EuiPanel
-              className={'dataSourceAggregatedViewOuiPanel'}
-              borderRadius="none"
-              hasBorder={false}
-              color="transparent"
-              paddingSize="s"
-            >
-              {titleComponent}
-            </EuiPanel>
+            {titleComponent}
```

Comment:
This is minor. I think the titleComponet should be outside EuiContextMenuPanel since it is using a EuiPopoverTitle. I feel like it is not part of the EuiContextMenuPanel

Reply:
Fixed.

Reply:
Thanks